mypy/exprtotype.py

```diff
@@ -61,7 +61,10 @@ def expr_to_unanalyzed_type(expr: Expression, _parent: Optional[Expression] = No
                 args = expr.index.items
             else:
                 args = [expr.index]
-            base.args = tuple(expr_to_unanalyzed_type(arg, expr) for arg in args)
+            if base.name == 'Annotated':
```

Comment:
is this a good way of checking the name? Looks like base here doesn't have a fullname